gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
There's still an encoding issue here.

Reply:
I cloned your branch locally and it looks fine so it is probably just some GitHub issue.